gtsam/linear/GaussianConditional.h

```diff
@@ -24,6 +24,7 @@
 #include <gtsam/global_includes.h>
 #include <gtsam/linear/JacobianFactor.h>
 #include <gtsam/inference/Conditional.h>
+#include <gtsam/inference/Conditional-inst.h>
```

Comment:
This should go to the cpp file

Reply:
So it looks like the -inst.h file will have to go in the header so that the subclasses can access the definition.